api/discord/controllers/discord.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  async discordAuth(ctx){
+    const { code } = ctx.query;
+    if (!code) {
+      return ctx.send({ message: 'There is not a code in the url' }, 500);
+    }
+    try {
+      // post a request to get back the token type and an access token
```

Comment:
usually, you don't say post a request. The proper way to say it post request. Same goes with all the other requests. For example, "I made a get request, and I also made a put request".